From 150004f62e929f760dfeb1f36c7545685505e979 Mon Sep 17 00:00:00 2001
From: Iouri Tarassov <iourit@linux.microsoft.com>
Date: Tue, 15 Feb 2022 18:11:52 -0800
Subject: [PATCH 01/48] drivers: hv: dxgkrnl: Add virtual compute device VMBus
 channel guids

Add VMBus channel guids, which are used by hyper-v virtual compute
device driver.

Signed-off-by: Iouri Tarassov <iourit@linux.microsoft.com>
---
 hyperv.h | 32 ++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/hyperv.h b/hyperv.h
new file mode 100644
index 000000000000..773a418c7a67
--- /dev/null
+++ b/hyperv.h
@@ -0,0 +1,32 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+/*
+ * Copyright (c) 2022, Microsoft Corporation.
+ *
+ * Author:
+ *   Iouri Tarassov <iourit@linux.microsoft.com>
+ *
+ */
+
+#ifndef _HYPERV_H
+#define _HYPERV_H
+
+#include <linux/hyperv.h>
+
+/*
+ * GPU paravirtualization global DXGK channel
+ * {DDE9CBC0-5060-4436-9448-EA1254A5D177}
+ */
+#define HV_GPUP_DXGK_GLOBAL_GUID \
+	.guid = GUID_INIT(0xdde9cbc0, 0x5060, 0x4436, 0x94, 0x48, \
+			  0xea, 0x12, 0x54, 0xa5, 0xd1, 0x77)
+
+/*
+ * GPU paravirtualization per virtual GPU DXGK channel
+ * {6E382D18-3336-4F4B-ACC4-2B7703D4DF4A}
+ */
+#define HV_GPUP_DXGK_VGPU_GUID \
+	.guid = GUID_INIT(0x6e382d18, 0x3336, 0x4f4b, 0xac, 0xc4, \
+			  0x2b, 0x77, 0x3, 0xd4, 0xdf, 0x4a)
+
+#endif
-- 
2.43.0

